Quiz/ViewController.swift

```diff
@@ -9,6 +9,17 @@ class ViewController: UIViewController {
     @IBOutlet var questionLabel: UILabel!
     @IBOutlet var answerLabel: UILabel!
     
+    let questions: [String] = [
+        "From what is cognac made?",
+        "What is 7+7?",
+        "What is the capital of Vermont?"
+    ]
+    let answers: [String] = [
+        "Grapes",
+        "14",
+        "Montpelier"
+    ]
+    
```

Comment:
Aaron - please re-read the assignment. 
I removed this data from the ViewController and put it in a new Model class Quiz.swift.
I'd like you to rewrite the ViewController to use this new model class

Reply:
See the TODO's list in the ViewController